### DISCUSSION TITLE
Roms not being recognised

### DISCUSSION THREAD

**Discussion by bouncingbenji** [Swordfish90/Lemuroid#507] [Q&A]
Good morning/evening 

New user here but when I scan my roms using lemuroid not all of them get found? But the missing ones work just fine in the standalone emulators? It's mostly with the snes and genesis? 

Also is it better to download from here or the playstore?

**Comment by cyruzwings** [comment]
> Good morning/evening 
> 
> New user here but when I scan my roms using lemuroid not all of them get found? But the missing ones work just fine in the standalone emulators? It's mostly with the snes and genesis? 
> 
> Also is it better to download from here or the playstore? 

did you put roms based on folder? gb/gba/nes/snes?
if compressed make sure its zip
